decidim-proposals/spec/system/participatory_texts_spec.rb

```diff
@@ -31,30 +29,66 @@ def should_have_proposal(selector, proposal)
   end
 
   context "when listing proposals in a participatory process as participatory texts" do
-    let!(:proposals) { create_list(:proposal, 3, :published, component: component) }
-
-    context "when voting is enabled" do
+    context "when admin has not yet imported a participatory text" do
       let!(:component) do
         create(:proposal_component,
                :with_participatory_texts_enabled,
-               :with_votes_enabled,
                manifest: manifest,
                participatory_space: participatory_process)
       end
 
-      it_behaves_like "lists all the proposals ordered"
+      before do
+        visit_component
+      end
+
+      it "renders an empty title" do
+        within ".heading2" do
+          expect(page).to have_content("")
```

Comment:
Remember to change the test too! 😄

Reply:
Thanks. I'll do a first commit of just the application code so you can take a look at the solution in the meanwhile.